static/js/src/advantage/offers/components/OffersList/OffersList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import * as Sentry from "@sentry/react";
+import useGetOffersList from "../../hooks/useGetOffersList";
+import Offer from "../Offer";
+import { Offer as OfferType } from "../../types";
+
+const OffersList = () => {
+  const { isLoading, isError, data: offersList, error } = useGetOffersList();
+
+  if (isError) {
+    Sentry.captureException(error);
+    return <p>Something went wrong while trying to retrieve your offers.</p>;
```

Comment:
Maybe offer the user since advise?

Reply:
@wgx should we ask them to reach out to the saleperson who added the offer to their account?

Reply:
When the user is not logged in they get this error when it would be good to let them know they need to sign in.

Reply:
Or maybe we should redirect to /advantage if the user is logged out?

Reply:
The is what we do for user accounts. Not a huge fan of that behaviour but it would be consistent.